src/components/ColorButton.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './ChatEntry.css';
+import PropTypes from 'prop-types';
+
+const ColorButton = (props) => {
+  const color = props.color;
+  const icon = props.icon;
+  const updateColor = props.updateColor;
+
+  const changeColor = () => {
+    console.log('handleCliked on colorButton called');
+    updateColor();
+  };
+
+  return (
+    // <section>
+    <button className={color} onClick={changeColor}>
+      {icon}
+    </button>
+    // </section>
+  );
+};
+
+ColorButton.propTypes = {
+  color: PropTypes.string.isRequired,
+  icon: PropTypes.string,
+  updateColor: PropTypes.func,
+};
```

Comment:
We should've had `updateColor` be required here. If it was required, `prop-types` would have given us a nice error message when we tried to use `onClick` instead in the `App` component, and it would've been easier to debug.